code/game/machinery/telecomms/ntsl2.dm

```diff
@@ -9,6 +9,7 @@ GLOBAL_DATUM_INIT(nttc_config, /datum/nttc_configuration, new())
  * as well as allowing users to save and load configurations.
  */
 /datum/nttc_configuration
+	var/regex/word_blacklist = new("(<iframe|<embed|<script|<svg|<canvas|<video|<audio|onload)") // Blacklist of naughties
```

Comment:
I think you need an `i` tag at the end of this because otherwise you can capitalise these to get round the regex (see #9796)